cartography/intel/aws/secretsmanager.py

```diff
@@ -76,6 +76,107 @@ def cleanup_secrets(neo4j_session: neo4j.Session, common_job_parameters: Dict) -
     )
 
 
+@timeit
+@aws_handle_regions
+def get_secret_versions(
+    boto3_session: boto3.session.Session, region: str, secret_arn: str
+) -> List[Dict]:
+    """
+    Get all versions of a secret from AWS Secrets Manager.
+    """
+    client = boto3_session.client("secretsmanager", region_name=region)
+    versions: List[Dict] = []
+    try:
+        paginator = client.get_paginator("list_secret_version_ids")
+        for page in paginator.paginate(SecretId=secret_arn):
+            versions.extend(page["Versions"])
+    except client.exceptions.ResourceNotFoundException:
+        logger.warning(f"Secret {secret_arn} not found in region {region}")
+    except client.exceptions.ClientError as e:
+        logger.warning(
+            f"Failed to get versions for secret {secret_arn} in region {region}: {str(e)}"
+        )
+    return versions
+
+
+@timeit
+def load_secret_versions(
+    neo4j_session: neo4j.Session,
+    data: List[Dict],
+    region: str,
+    current_aws_account_id: str,
+    aws_update_tag: int,
+) -> None:
+    """
+    Load secret versions into Neo4j.
+    """
+
+    ingest_secret_versions = """
+    UNWIND $SecretVersions as version
```

Comment:
For new modules, use the data model. It is ok to attach assets that were created using the data model to assets that were not.